IFC4x3/Properties/g/GroundingType_0GRwj6SjTAtwY3611gVZUL/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GroundingType_0GRwj6SjTAtwY3611gVZUL" Name="GroundingType" UniqueId="106fab46-72d7-4adf-a883-18106a7e3795" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel" />
```

Comment:
Penum seems to be missing.

Reply:
Changed to single value. Commit : b5e658cad21e4944423ed95d945268c722299c68